src/utils/fetchData.js

```diff
@@ -2,7 +2,9 @@ import queryPerfData from '@mozilla-frontend-infra/perf-goggles';
 import { BENCHMARKS, CONFIG } from '../config';
 import prepareData from './prepareData';
 
-const fetchData = async (platform, benchmark) => {
+const convertToMs = value => (value * 24 * 3600);
```

Comment:
It's no miliseconds but seconds.

Reply:
Riiiight, thank you for pointing it out. I meant to ask you about this - can I create a general purpose utils.js file from which I could export every little method I find reusable?

Reply:
Sure thing!